test/modules/packages/collections/CollectionBulk.compopts

```diff
@@ -0,0 +1,3 @@
+-sisDeque --baseline
+-sisBag --no-loop-invariant-code-motion
+-sisBoundedDeque --baseline
```

Comment:
Why do we need `--baseline` here? If it is b/c of a bug, I think you should a reference to it, otherwise, let's remove it.

Reply:
I believe I used `--baseline` as I was experiencing some issues with non-deterministic behavior even with just '--no-loop-invariant-code-motion', but now that I've re-tested it, it seems to work pretty well. I believe the issue had to do with `fifo` tasking layer and I forgot to remove it, fixing it right now. (Gonna make these fixes in small incremental patches).